services/src/performance-tests/src/main/groovy/org/openkilda/performancetests/helpers/TopologyHelper.groovy

```diff
@@ -40,25 +31,26 @@ class TopologyHelper extends org.openkilda.functionaltests.helpers.TopologyHelpe
     List<String> statControllers
 
     CustomTopology createRandomTopology(int switchesAmount, int islsAmount) {
+        if (islsAmount < (switchesAmount -1)) {
+            throw new Exception("Not enough ISLs were requested. islsAmount should be >= switchesAmount")
```

Comment:
I thought I can request 3 switches and 2 isls, why not?

Reply:
I am with you on it.
fixed